Doc/library/cmath.rst

```diff
@@ -322,7 +322,7 @@ Constants
 
 .. data:: inf
 
-   Floating-point positive infinity. Equivalent to ``float('inf')``.
+   Floating-point positive infinity. Equivalent to :data:`math.inf`.
```

Comment:
```suggestion
   Floating-point positive infinity. Equivalent to ``float('inf')``.
```
Lets revert it, this not adds something.

Reply:
As with nan, data cmath.inf is more similar to data math.inf than function call float('inf').  I prefer the change.